opba-banking-rest-impl/src/main/java/de/adorsys/opba/tppbankingapi/controller/TppBankingApiPisInfoController.java

```diff
@@ -0,0 +1,72 @@
+package de.adorsys.opba.tppbankingapi.controller;
+
+import de.adorsys.opba.protocol.api.dto.context.UserAgentContext;
+import de.adorsys.opba.protocol.api.dto.request.FacadeServiceableRequest;
+import de.adorsys.opba.protocol.facade.dto.result.torest.FacadeResult;
+import de.adorsys.opba.protocol.facade.services.pis.GetPaymentInformationService;
+import de.adorsys.opba.protocol.facade.services.pis.GetPaymentStatusService;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoBody;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentInfoRequest;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentStatusBody;
+import de.adorsys.opba.protocol.api.dto.request.payments.PaymentStatusRequest;
+import de.adorsys.opba.restapi.shared.mapper.FacadeResponseBodyToRestBodyMapper;
+import de.adorsys.opba.restapi.shared.service.FacadeResponseMapper;
+import de.adorsys.opba.tppbankingapi.Const;
+import de.adorsys.opba.tppbankingapi.pis.model.generated.PaymentInitiationWithStatusResponse;
+import de.adorsys.opba.tppbankingapi.pis.resource.generated.TppBankingApiPaymentStatusPisApi;
+import lombok.RequiredArgsConstructor;
+import org.mapstruct.Mapper;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import static de.adorsys.opba.restapi.shared.GlobalConst.SPRING_KEYWORD;
+
+@RestController
+@RequiredArgsConstructor
+public class TppBankingApiPisInfoController implements TppBankingApiPaymentStatusPisApi {
+    private final GetPaymentStatusService paymentStatusService;
+    private final GetPaymentInformationService paymentInfoService;
+    private final UserAgentContext userAgentContext;
+    private final FacadeResponseMapper mapper;
+    private final PaymentInfoBodyToApiMapper paymentInfoResponseMapper;
+    private final PaymentStatusBodyToApiMapper paymentStatusResponseMapper;
+
+
+    @Override
+    public CompletableFuture getPaymentInformation(UUID xRequestID) {
+        return paymentInfoService.execute(
+                PaymentInfoRequest.builder()
+                        .facadeServiceable(FacadeServiceableRequest.builder()
+                                // Get rid of CGILIB here by copying:
+                                .uaContext(userAgentContext.toBuilder().build())
```

Comment:
This request doesn't seem to need uaContext as ut is FinTech originating request without further consent authorization. Please check if we can remove 'uaContext' here